spynnaker/pyNN/models/utility_models/synapse_expander/synapse_expander.py

```diff
@@ -33,7 +36,7 @@
 
 def synapse_expander(
         app_graph, placements, transceiver, provenance_file_path,
-        executable_finder):
+        executable_finder, extract_iobuf, no_sync_changes):
```

Comment:
This seems to take the number of sync changes just so it can be ignored later on.